src/samples/dolphin/objects/player.cpp

```diff
@@ -85,4 +86,9 @@ void Player::update(Pad &t_pad)
     mesh.position.x +=
         Math::sin(mesh.rotation.z) * velocity * (isJumping + 1);
     mesh.position.y += lift;
+}
+
+u8 Player::getIsJumping()
```

Comment:
It sounds weird, shouldnt be this "`isJumping()`"? You can change your var to `bIsJumping` (as you did before) or `isJumping_`  + inline